API/JSON:API.md

```diff
@@ -0,0 +1,14 @@
+JSON:API is a specification for tailoring APIs that respond in JSON. Its purpose is to take the API `design` out of the equation so that developers can focus on API `delivery`. Due to its structure, tooling can be built around the specification which handles the bulk of work regarding parsing requests and formatting responses.
```

Comment:
```suggestion
JSON:API is a specification for tailoring APIs that respond with JSON. Its purpose is to take the API `design` out of the equation so that developers can focus on API `delivery`. Due to its structure, tooling can be built around the specification for handling the bulk of the work regarding parsing requests and formatting responses.
```